plugins/badges/src/components/EntityBadgesDialog.tsx

```diff
@@ -43,19 +43,10 @@ type Props = {
   entity: Entity;
 };
 
-const useStyles = makeStyles({
-  codeBlock: {
-    '& code': {
-      whiteSpace: 'pre-wrap',
-    },
-  },
-});
```

Comment:
Note removed all of the useStyles/classes formatting and such as when I did, there did not look to be any visual difference, so I believe the `<CodeSnippet>` component already drove the appropriate formatting.